src/sql/workbench/browser/designer/media/designer.css

```diff
@@ -152,3 +155,34 @@
 	padding-top: 10px;
 	padding-left: 25px;
 }
+
+.designer-component .codicon-grabber {
+	cursor: move;
+}
+
+.designer-component .slick-cell.isDragging {
```

Comment:
we should avoid having colors defined in css directly, especially in newly introduced files, please follow the examples in designer and set colors by theme colors.

Reply:
We don't have access to the HTMLElement like the other examples, since we only have access to the cells via Slickgrid. The official way to apply any style to SlickGrid cell is to use `setCellCssStyles` (https://github.com/microsoft/azuredatastudio/pull/19154/files#diff-5efde37ccdf9b53583ac82c8397d27e664bef5afbc425d8d44da27460d1aaee3R84) and then add styling for that class.

Reply:
you should use registerThemingParticipant:
for example: https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/browser/designer/designerIssuesTabPanelView.ts#L57